src/util/dnsbatch/brute_selector_server.py

```diff
@@ -0,0 +1,300 @@
+                            result_data = {"domain": domain, "selector": selector, "value": value}
+                            active_sessions[self.session_id].append(result_data)
+                            results.append(result_data)
+                        elif http:
+                            results.append({"domain": domain, "selector": selector, "value": value})
+                except Exception as e:
+                    print(f"Error resolving {selector}: {e}")
+        
+        time_taken = time.time() - start_time
+        print(f"Block {block_id}: Completed in {time_taken:.2f} seconds ({len(results)} valid)")
+        
+        if not http and self.session_id in active_sessions:
+            socketio.emit("blockProcessingCompleted", {
+                "domain": domain,
+                "blockId": block_id,
+                "timeTaken": time_taken,
+                "validCount": len(results)
+            }, room=self.session_id)
+            
+        return results
+
+    def _emit_results_background(self):
+        if self.session_id not in active_sessions:
+            return
+            
+        while self.session_id in active_sessions:
+            results = active_sessions[self.session_id]
+            if len(results) > 0:
+                selectorResult = results.pop(0)                
+                try:
+                    socketio.emit("bruteDomainResponse", selectorResult, room=self.session_id)
+                except Exception as e:
+                    print(f"Error emitting result: {e}")
+                    break
+                time.sleep(0.1)
+            else:
+                time.sleep(0.5)
+                
+    def _add_to_database(self, domain, selector):
+        data = {
+            "domain": domain,
+            "selector": selector,
+            }
+        try:
+            response = requests.post(ARCHIVE_API_URL, json=data)
+        except requests.exceptions.RequestException as e:
+            print(f"Request failed: {e}")
+
+    def find_valid_selectors(self, 
+                           domain: str,
+                           selectors: Union[List[str], str],
+                           verbose: bool = False,
+                           http: bool = True) -> List[str]:
+        start_time = time.time()
+        
+        emitter_thread = None
+        if not http and self.session_id in active_sessions:
+            emitter_thread = threading.Thread(target=self._emit_results_background)
+            emitter_thread.daemon = True
+            emitter_thread.start()
+
+        if isinstance(selectors, str):
+            with open(selectors, 'r') as file:
+                selector_list = [line.strip() for line in file.readlines()]
+        else:
+            selector_list = selectors
+        unique_selectors = list(set(selector for selector in selector_list if selector))
+        
+        if verbose:
+            # print(f"Using {self.PROCESS_COUNT} processes")
```

Comment:
Here is a python syntax error under  `if verbose:` because the scope is empty (commented out), so the file won't run at all